package.json

```diff
@@ -28,7 +28,8 @@
     "nextra": "latest",
     "nextra-theme-docs": "latest",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-loader-spinner": "^6.1.6"
```

Comment:
There is a material-ui spinner, lets use that instead.

Reply:
Done, idk why I was using react-spinner anyway, just saw it on one of the tutorial videos I have seen lol